### ISSUE TITLE
IG,Dm

### ISSUE THREAD

**Issue by chesmix4** [ifedapoolarewaju/igdm#205]
When I use the IG:dm, I saw that the person i was talking to sent some texts to me by looking on my phone, but later on, checking on the IG:Dm, I didnt see the texts the person sent, only mine.

**Comment by chesmix4** [comment]
Woah, fast responce, I like it. Well no, I tried just deleting the tab and
then starting it up again, but no. It seems that when Its not open and the
other person sends texts, it shows, but when the program is off, It
doesnt show the texts sent beforehand.

**Comment by ifedapoolarewaju** [comment]
>  It seems that when Its not open and the
other person sends texts, it shows, but when the program is off, It
doesnt show the texts sent beforehand.

I'm a bit confused by this. Could you please rephrase?

**Comment by chesmix4** [comment]
Lets see, so when I received a text from this person, my computer was off
(therefore the program obviously wasn't running). I replied to this person
on my phone with the real Instagram App. When i came back home, I ran the
program and saw that the texts that the person sent was not there, but my
responce that I gave was there. After a while, this person had sent another
text and this time, while the program was running, i received the text.
